src/app/_components/Table/TooltipHeader.tsx

```diff
@@ -31,20 +35,20 @@ export function TooltipHeader({
             aria-describedby={tooltipId}
             className={clsx(
               'group focus-visible:brand-outline',
-              TOUCH_TARGET.class,
-              TOUCH_TARGET.offsetClass,
+              TOUCH_TARGET.touchAreaPadding,
+              TOUCH_TARGET.touchAreaOffset,
             )}
           >
             <span
               className={clsx(
-                TOUCH_TARGET.iconWrapperPadding,
+                TOUCH_TARGET_ICON.touchAreaPadding,
                 'flex items-center justify-center rounded-full transition ease-in-out',
                 open
                   ? 'bg-brand-600/70 text-brand-200'
                   : 'text-brand-400 group-hover:bg-brand-700 group-hover:text-brand-300',
               )}
             >
-              <Icon component={Question} size={TOUCH_TARGET.iconSize} />
+              <Icon component={Question} size={20} />
```

Comment:
This could be moved to TOUCH_TARGET_ICON as `visibleElementSize` but we usually directly pass the value to the component, so I kept it this way. Let me know if you want it as `visibleElementSize` in TOUCH_TARGET_ICON 🙂

Reply:
Yes! I think we can use `visibleElementSize` in this case. Thank you.